### PR TITLE
In-service morning

### PR DESCRIPTION
**Pull request by BorePlusPlus**
When morning command is run within service directory, it runs limited to the service and it's dependencies rather than entire workspace.

### CODE REVIEW

**Comment by cliftonc** [comment]
lgtm

File: commands/install.js [hunk 1]
```diff
@@ -66,7 +68,18 @@ function cmd(bosco, args, next) {
...
+
+  function setRunRepos(cb) {
+    if (CmdHelper.checkInService(bosco)) {
+      RunListHelper.getRepoRunList(bosco, bosco.getRepos(), repoRegex, '$^', null, function(err, runRepos) {
+        repos = runRepos;
+        cb(err);
+      });
+    } else {
+      cb();
+    }
```
Comment (geophree) [inline commands/install.js]: Early exit, please.  Avoid the pyramid:
```js
if (!CmdHelper.checkInService(bosco)) return cb();

RunListHelper.getRepoRunList(bosco, bosco.getRepos(), repoRegex, '$^', null, function(err, runRepos) {
  repos = runRepos;
  cb(err);
});
```

File: commands/pull.js [hunk 2]
```diff
@@ -114,7 +115,18 @@ function cmd(bosco, args, next) {
...
+
+  function setRunRepos(cb) {
+    if (CmdHelper.checkInService(bosco)) {
+      RunListHelper.getRepoRunList(bosco, bosco.getRepos(), repoRegex, watchNothing, null, function(err, runRepos) {
+        repos = runRepos;
+        cb(err);
+      });
+    } else {
+      cb();
+    }
```
Comment (geophree) [inline commands/pull.js]: Early exit please.

**Pull request merged**